cram_pr2/cram_pr2_shopping_demo/src/plans.lisp

```diff
@@ -0,0 +1,173 @@
+(in-package :cram-pr2-shopping-demo)
+
+(defparameter *table* (cl-transforms-stamped:make-pose-stamped
+                        "map" 0.0
+                        (cl-transforms:make-3d-vector -3.1 0.7 0.75)
+                        (cl-transforms:make-identity-rotation)))
+
+(defparameter *pose-grasping* (cl-transforms-stamped:make-pose-stamped
+                               "map" 0.0
+                               (cl-transforms:make-3d-vector -1.5 -0.3 0)
+                               (cl-transforms:axis-angle->quaternion
+                                (cl-transforms:make-3d-vector 0 0 1) (/ pi -2))))
+
+(defparameter *pose-grasping-left* (cl-transforms-stamped:make-pose-stamped
+                               "map" 0.0
+                               (cl-transforms:make-3d-vector -1 -0.35 0)
+                               (cl-transforms:axis-angle->quaternion
+                                (cl-transforms:make-3d-vector 0 0 1) (/ pi -2))))
+
+(defparameter *pose-grasping-right* (cl-transforms-stamped:make-pose-stamped
+                               "map" 0.0
+                               (cl-transforms:make-3d-vector -2 -0.35 0)
+                               (cl-transforms:axis-angle->quaternion
+                                (cl-transforms:make-3d-vector 0 0 1) (/ pi -2))))
+
+
+(defparameter *pose-near-table* (cl-transforms-stamped:make-pose-stamped
+                                  "map" 0.0
+                                  (cl-transforms:make-3d-vector -2.5 0.5 0)
+                                  (cl-transforms:axis-angle->quaternion
+                                   (cl-transforms:make-3d-vector 0 0 1) (/ pi 1))))
+
+(defparameter *looking-pose-mid* (cl-transforms-stamped:make-pose-stamped
+                                  "map" 0.0
+                                  (cl-transforms:make-3d-vector -1.5 -1.05 1.15)
+                                  (cl-transforms:make-identity-rotation)))
+
+(defparameter *pose-detecting* (cl-transforms-stamped:make-pose-stamped
+                               "map" 0.0
+                               (cl-transforms:make-3d-vector -1.5 1 0)
+                               (cl-transforms:axis-angle->quaternion
+                                (cl-transforms:make-3d-vector 0 0 1) (/ pi -2))))
+
+(defun move-object (?object)
+    (let ((?grasping-arm :right)
+          (?grasp-pose *pose-grasping*)
+          (?grasp-pose-left *pose-grasping-left*)
+          (?grasp-pose-right *pose-grasping-right*)
+          (?pose-near-table *pose-near-table*)
+          (?table *table*)
+          (map->obj NIL)
+          (map->base NIL)
+          (trans NIL))
+
+      (exe:perform 
+       (desig:a motion (type moving-torso)
```

Comment:
IMO it is easier to read a designator when the key value pairs start on a new line:
```
(desig:a motion
         (type moving-torso)
         (joint-angle 0))
```